docs/reference/seismometer.rst

```diff
@@ -9,9 +9,14 @@ Public API
 .. autosummary::
    :toctree: api/
 
+   ExploreModelEvaluation
```

Comment:
Consideration:  I almost think these deserve their own section.  This is becoming _the_ public api more than the old functions.  
Could also be a followup PR, as I think we'll want some dedicated guide section(s) the control+plot pattern

Reply:
FINC